### PR TITLE
Support routed feeds and add new feeds in config

### PR DESCRIPTION
**Pull request by gabaldon**

### CODE REVIEW

File: packages/ui/utils/getDataFeedStatus.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+    const updateLimitTimestamp = new Date().getTime() - Number(heartbeat)
+    const lastResultTimestampMilliseconds = Number(`${lastResultTimestamp}000`)
+    if (updateLimitTimestamp > lastResultTimestampMilliseconds) {
+      return dataFeedStatus.delay
+    } else {
+      return dataFeedStatus.operational
+    }
   } else {
     return dataFeedStatus.operational
   }
```
Comment (Tommytrg) [inline packages/ui/utils/getDataFeedStatus.js:14]: if-else hell :fire: 

```suggestion
  if (heartbeat) {
    const updateLimitTimestamp = new Date().getTime() - Number(heartbeat)
    const lastResultTimestampMilliseconds = Number(`${lastResultTimestamp}000`)
    if (updateLimitTimestamp > lastResultTimestampMilliseconds) {
      return dataFeedStatus.delay
    }
  }
  return dataFeedStatus.operational
```
```

File: packages/ui/components/DataFeeds.vue [hunk 2]
```diff
@@ -75,7 +75,9 @@ export default {
               value: feed.lastResult,
               lastResultTimestamp: feed.lastResultTimestamp || '0',
               label: feed.label,
-              timeToUpdate: Number(feed.heartbeat) + Number(feed.finality),
+              timeToUpdate: feed.heartbeat
```
[the remark this answers is not in the document]
Comment (gabaldon) [reply packages/ui/components/DataFeeds.vue:78]: There is always a finality

File: packages/ui/locales/de.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
```
Comment (guidiaz) [inline packages/ui/locales/de.json:14]: really needed on Deutche locale ?

File: packages/ui/locales/es.json [hunk 4]
```diff
@@ -10,7 +10,9 @@
...
-    "feed_description": "Esto es un data feed de Witnet para {name}, disponible en {network}. El último resultado disponible es {value} (actualizado el {date}). Este data feed se actualiza cada {heartbeat}, o cada vez que su valor varía más de un {deviation}."
+    "feed_description": "Esto es un data feed de Witnet para {name}, disponible en {network}. El último resultado disponible es {value} (actualizado el {date}). Este data feed se actualiza cada {heartbeat}, o cada vez que su valor varía más de un {deviation}.",
+    "feed_description_routed": "Esto es un price feed de Witnet para {name}, disponible en {network}. El último resultado disponible es {value} (actualizado el {date}). Este price feed se actualizará en el momento en el que se actualize alguno de los price feeds relacionados.",
+    "feed_description_routed_cached": "Esto es un price feed de Witnet para {name}, disponible en {network}. El último resultado disponible es {value} (actualizado el {date}). Este price feed se actualizará cada {heartbeat} o cada vez que se actualize alguno de los price feeds relacionados."
```
Comment (guidiaz) [inline packages/ui/locales/es.json:15]: "de los que depende" en lugar de "relacionados"

File: packages/ui/locales/fr.json [hunk 5]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
   },
```
Comment (guidiaz) [inline packages/ui/locales/fr.json:15]: really needed on French locale ?

File: packages/ui/locales/hi.json [hunk 6]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
```
Comment (guidiaz) [inline packages/ui/locales/hi.json:14]: really needed on Hindi locale ?

File: packages/ui/locales/ja.json [hunk 7]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
```
Comment (guidiaz) [inline packages/ui/locales/ja.json:14]: really needed on Japanese locale ?

File: packages/ui/locales/ko.json [hunk 8]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
```
Comment (guidiaz) [inline packages/ui/locales/ko.json:14]: really needed on Korean locale ?

File: packages/ui/locales/pt.json [hunk 9]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
```
Comment (guidiaz) [inline packages/ui/locales/pt.json:14]: really needed on Portuguese locale ?

File: packages/ui/locales/ru.json [hunk 10]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
+    "feed_description_routed": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}."
```
Comment (guidiaz) [inline packages/ui/locales/ru.json:14]: really needed on Rusian locale ?

File: packages/ui/locales/zh.json [hunk 11]
```diff
@@ -10,7 +10,8 @@
     "underlying_feed_contract": "Underlying feed contract address",
     "erc2362_asset_id": "ERC2362 asset ID",
     "integration_details_description": "Witnet price feeds can be integrated into your own {0} contracts in two different ways:",
-    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}."
+    "feed_description": "This is a Witnet powered {name} data feed available on {network}. Last reported value is {value} (updated on {date}). This data feed is updated every {heartbeat}, or every time the price changes by more than {deviation}.",
```
Comment (guidiaz) [inline packages/ui/locales/zh.json:13]: really needed on Chinese locale ?

File: packages/api/test/feeds.spec.ts [hunk 12]
```diff
@@ -26,6 +26,7 @@ const state: {
   testClient: null,
   server: null
 }
+
```
Comment (Tommytrg) [inline packages/api/test/feeds.spec.ts:29]:
```suggestion
```

File: packages/ui/components/DataFeedDetails.vue [hunk 13]
```diff
@@ -13,14 +13,18 @@
...
       :feed-name="normalizedFeed.name"
       :network-name="normalizedFeed.networkName"
       :last-result-value="lastResultValue"
       :last-result-date="lastResultDate"
       :feed-time-to-update="feedTimeToUpdate"
       :deviation="normalizedFeed.deviation"
     />
-    <Fieldset :title="$t('data_feed_details.trigger_parameters')">
+    <Fieldset
+      v-if="maxTimeToResolve || normalizedFeed.deviation"
```
Comment (Tommytrg) [inline packages/ui/components/DataFeedDetails.vue:25]: Is this equivalent to check if it's routed?
```suggestion
      v-if="!normalizedFeed.isRouted"
```

Comment (gabaldon) [reply packages/ui/components/DataFeedDetails.vue:25]: it is not, it could be a routed data feed with a heartbeat

**Pull request merged**